README.md

```diff
@@ -0,0 +1,407 @@
+
+<details>
+  <summary>Manager's and Admin's Endpoints: Add, update, and delete books.</summary>
+
+  - **Add a new book**
+    - Request `POST /api/books`
+    - Response: `201 Created`
+  - **Update a book**
+    - Request `PUT /api/books/{id}`
+    - Response: `202 Accepted`
+  - **Delete a book**
+    - Request `DELETE /api/books/{id}`
+    - Response: `204 No Content`
+
+</details>
+
+### 📑Category
+
+<details>
+  <summary>User's Endpoints: View categories.</summary>
+
+  - **Get all categories**
+    - Request `GET /api/categories`
+    - Response: `200 OK`
+  - **Get a category by ID**
+    - Request `GET /api/categories/{id}`
+    - Response: `200 OK`
+  - **Get all books from category**
+    - Request `GET /api/categories/{id}/books`
+    - Response: `200 OK`
+
+</details>
+
+<details>
+  <summary>Manager's and Admin's Endpoints: Create, update, and delete book categories.</summary>
+
+  - **Add a new category**
+    - Request `POST /api/categories`
+    - Response: `201 Created`
+  - **Update a category**
+    - Request `PUT /api/categories/{id}`
+    - Response: `202 Accepted`
+  - **Delete a category**
+    - Request `DELETE /api/categories/{id}`
+    - Response: `204 No Content`
+
+</details>
+
+### 🛒Shopping Cart
+
+<details>
+  <summary>User's Endpoints: Add books to the cart, view cart items, update and remove items from the cart.</summary>
+
+  - **Get current user's shopping cart**
+     - Request `GET /api/cart`
+     - Response: `200 Ok`
+  - **Add a book to the cart**
+     - Request `POST /api/cart`
+     - Response: `201 Created`
+  - **Update book quantity**
+     - Request `PUT /api/cart/cart-items/{cartItemId}`
+     - Response: `202 Accepted`
+  - **Remove a book from the cart**
+     - Request `DELETE /api/cart/cart-items/{cartItemId}`
+     - Response: `204 No Content`
+
+</details>
+
+### 🛍️Order
+
+<details>
+  <summary>User's Endpoints: Complete the purchase, view past orders and their details.</summary>
+
+- **Create order**
+       - Request `POST /api/orders`
+       - Response: `201 Created`
+    - **Get all orders for current user**
+       - Request `GET /api/orders`
+       - Response: `200 Ok`
+    - **Get an order details by ID**
+     - Request `GET /api/orders/{orderId}/items`
+     - Response: `200 Ok`
+  - **Get an ordered book details by ID**
+       - Request `GET /api/orders/{orderId}/items/{id}`
+        - Response: `200 Ok`
+
+</details>
+
+<details>
+  <summary>Manager's and Admin's Order Endpoints: Update order statuses.</summary>
+
+- **Update order status**
+       - Request `PATCH /api/orders/{id}`
+       - Response: `202 Accepted`
+
+</details>
+
+***
+
+## 📺Short Overview
+
+<a href="https://www.loom.com/share/0e9df6c8dab44ffcbb104629e2afa5ce">
+  <img style="max-width:200px;" src="img/loom.png" alt="Backend for Online Bookstore - Watch Video">
+</a>
+
+***
+
+## 🚀Setup Instructions
+
+### 🧰Required
+
+- **Java 21+**
+- **Maven 3+**
+- **MySQL 8+**
+- **Docker**
+- **Docker Compose**
```

Comment:
the thing is, docker is there exactly to not install any of that :) so they are not really a requirements bit more of a technologies used, but I think you already have that mentioned